steps/delius/event/find-events.ts

```diff
@@ -29,3 +31,22 @@ export async function isInEventContext(page: Page, crn: string, eventNumber: num
         (await eventOverview.first().textContent()).startsWith(`Event:${eventNumber.toString()}`)
     )
 }
+
+export const verifyKeyDates = async (page: Page, crn: string, eventNumber: number, date: Date) => {
+    await findCustodyForEventByCRN(page, crn, eventNumber)
+    await refreshUntil(page, () => verifyDate(page, 'Sentence Expiry Date:', date))
+    await verifyDate(page, 'Sentence Expiry Date:', date)
+    await verifyDate(page, 'Licence Expiry Date:', date)
+    await verifyTableDate(page, 'Parole Eligibility Date', date)
+    await verifyTableDate(page, 'Auto-Conditional Release Date', date)
+}
+
+const verifyDate = async (page: Page, label: string, date: Date) => {
+    const id = await page.locator('label', { hasText: label }).getAttribute('for')
+    await expect(page.locator(`[id="${id}"]`)).toHaveText(DeliusDateFormatter(date))
```

Comment:
Playwright recently added a method for this:
```suggestion
    await expect(page.getByLabel(label).toHaveText(DeliusDateFormatter(date))
```
See https://playwright.dev/docs/input#text-input